docs/components/Playground/editor.tsx

```diff
@@ -21,7 +21,7 @@ const Editor: React.FC<Props> = ({ code }) => {
   const copyHandler = (event: React.MouseEvent) => {
     event.stopPropagation();
     event.preventDefault();
-    // copy(code);
+    copy(code);
     // setToast({ text: isChinese ? "代码已拷贝至剪切板。" : "code copied." });
```

Comment:
Remove comment or add `//TODO: ` if it is a todo case

Reply:
Will be removed in #75